homeassistant/components/risco/binary_sensor.py

```diff
@@ -3,23 +3,76 @@
 from __future__ import annotations
 
 from collections.abc import Mapping
+from itertools import chain
 from typing import Any
 
 from pyrisco.cloud.zone import Zone as CloudZone
+from pyrisco.common import System
 from pyrisco.local.zone import Zone as LocalZone
 
 from homeassistant.components.binary_sensor import (
     BinarySensorDeviceClass,
     BinarySensorEntity,
+    BinarySensorEntityDescription,
 )
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from . import LocalData, RiscoDataUpdateCoordinator, is_local
-from .const import DATA_COORDINATOR, DOMAIN
+from .const import DATA_COORDINATOR, DOMAIN, SYSTEM_UPDATE_SIGNAL
 from .entity import RiscoCloudZoneEntity, RiscoLocalZoneEntity
 
+SYSTEM_ENTITY_DESCRIPTIONS = [
+    BinarySensorEntityDescription(
+        key="low_battery_trouble",
+        name="Low battery trouble",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+    ),
+    BinarySensorEntityDescription(
+        key="ac_trouble",
+        name="A/C trouble",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+    ),
+    BinarySensorEntityDescription(
+        key="monitoring_station_1_trouble",
+        name="Monitoring station 1 trouble",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+    ),
+    BinarySensorEntityDescription(
+        key="monitoring_station_2_trouble",
+        name="Monitoring station 2 trouble",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+    ),
+    BinarySensorEntityDescription(
+        key="monitoring_station_3_trouble",
+        name="Monitoring station 3 trouble",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+    ),
+    BinarySensorEntityDescription(
+        key="phone_line_trouble",
+        name="Phone line trouble",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+    ),
+    BinarySensorEntityDescription(
+        key="clock_trouble",
+        name="Clock trouble",
+        device_class=BinarySensorDeviceClass.PROBLEM,
+    ),
+    BinarySensorEntityDescription(
+        key="box_tamper",
+        name="Box tamper",
+        device_class=BinarySensorDeviceClass.TAMPER,
+    ),
+    BinarySensorEntityDescription(
+        key="programming_mode",
+        name="Programming mode",
+        icon="mdi:home-edit",
+    ),
```

Comment:
Please add entity translations and add icon translations

Reply:
Since these are the names by Risco, does it make sense to translate them?

Reply:
I think it still adds to the user experience. When a French company creates an integration we also translate the names to English in the code and back to French again via translations. Otherwise your English dashboard would be polluted by French names. I think the same can be said the other way around here.